veevavault/1.3/modules/ROOT/pages/veevavault-connector-config-topics.adoc

```diff
@@ -806,24 +222,24 @@ The following are the error codes that are caught in the Error Handling componen
 [[streaming_and_pagination]]
 == Streaming and Pagination
 
-All the operations in the connector (except Download Document) return an InputStream as a payload with respective results based on operation output. Because of this, by default, Mule applies Streaming Strategies. See xref:mule-runtime::streaming-about.adoc[Mule Streaming Strategies] for more details. The streaming strategies configuration fields are in the *Advanced* tab of the connector operations.
+All the operations in the connector (except *Download Document*) return an InputStream as a payload with respective results based on operation output. Because of this, by default, Mule applies Streaming Strategies. See xref:mule-runtime::streaming-about.adoc[Mule Streaming Strategies] for more details. The streaming strategies configuration fields are in the *Advanced* tab of the connector operations.
 
 image::veevavault-connector-streaming-pagination.png[image,width=601,height=124]
 
 The following operations in the connector provide a pagination mechanism based on Mule standard pagination.
 
-* <<get_documents,Get Documents>>
-* <<get_object_records,Get Object Records>>
-* <<query,Query>>
-* <<get_audit_details,Get Audit Details>>
+* *Get Documents*
+* *Get Object Records*
+* *Query*
+* *Get Audit Details*
 
-While using the paginated operations make sure to place a `For-Each/Splitter` element to retrieve each object (metadata is in JSON format) at a time. The pagination operations have *Fetch Size* and *Batch Size* fields.
+While using the paginated operations, make sure to place a `For-Each/Splitter` element to retrieve each object (metadata is in JSON format) at a time. The pagination operations have *Fetch Size* and *Batch Size* fields.
```

Comment:
I can think of many alternatives to this wording, based on what you precisely intend. 
Is the audience "using" or "configuring" one (or more) paginated operation by including (rather than "placing"?) one (or more) For-Each/Splitter element to retrieve one object at a time (rather than many?)? And are we saying that objects are in JSON format? Is that necessary to mention in this context, and why? And how does the FE/S element relate to the two fields?

Reply:
Deleted the part about JSON format since it is included in the next bullet, and replaced "place" with "include"